src/Development/Rattle/Profile.hs

```diff
@@ -0,0 +1,252 @@
+{-# LANGUAGE ScopedTypeVariables, RecordWildCards, TupleSections, ViewPatterns #-}
```

Comment:
It's new code, so feel free to play with it as you wish. I have made some suggestions, but they're all at the superficial level of Haskell suggestions, nothing actual content-based. Feel free to play around with it for your purposes, and don't feel obliged to make any of the changes.

Reply:
I think I have made all of your suggested changes.